### PR TITLE
#402 add genesis authority

### PR DESCRIPTION
**Pull request by chb0github**
I have the private key in PEM - TBD how to distribute

### CODE REVIEW

**Comment by prince-chrismc** [comment]
Since this is an example can we just commit it?

**Comment by chb0github** [comment]
Sure. I will submit a pyrsia PR right away

File: src/blockchain/src/blockchain.rs [hunk 1]
```diff
@@ -88,6 +88,12 @@ impl Debug for Blockchain {
 impl Blockchain {
     pub fn new(keypair: &identity::ed25519::Keypair) -> Self {
         let local_id = PeerId::from(identity::PublicKey::Ed25519(keypair.public()));
+        let genesis_pub_key: [u8; 44] = [
```
Comment (fishseabowl) [inline src/blockchain/src/blockchain.rs:91]: where is the private key of this pub key?

Comment (prince-chrismc) [reply src/blockchain/src/blockchain.rs:91]: Dump where is this being shared?

**Pull request merged**